.github/workflows/e2e-test.yml

```diff
@@ -32,7 +43,8 @@ jobs:
         echo "export const config = $PROJECT_CONFIG; export const testAccount = $TEST_ACCOUNT" > firebase-config.js
     - name: Yarn install
       run: |
-        yarn add firebase
+        echo "Installing firebase@${{ github.event.inputs.tag }}"
+        yarn add firebase@${{ github.event.inputs.tag }}
```

Comment:
Can we just make version number a required field and use the version number? Perhaps it should accept both version number and tag name, and be renamed to `versionOrTag`.